data.json

```
@@ -1253,4 +1253,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/53567221/129094169-e012e3fb-3ba9-47e7-9173-0e09e7517034.png",
     "issueNumber": 642
   }
```

Oh, you forgot a comma here.
```suggestion
  },
```